### DISCUSSION TITLE
important question ?

### DISCUSSION THREAD

**Discussion by sensormatic1985** [taligentx/dscKeybusInterface#228] [dscKeybusInterface Discussion]
Good afternoon, it is possible to add some lines to your code to be able to activate two relays to do or follow: take the signal from the alarm siren output when there is already an activation to be able to control two relays to condition them for different on-off uses, to lights, 110v sirens, strobe light, etc.

**Comment by Dilbert66** [comment]
These functions are best handled from the main home control software you use such as home assistant.   The dsc interface code's main job is to provide access to all inputs and outputs and send them to the home control software which can use this information to control actions  such as turn on multiple relays, send sms messages, turn on lights, etc .  No need to clutter up the arduino/esp code with unecessary logic.

**Reply by sensormatic1985** [reply]
OK thank you

**Comment by kricon** [comment]
If I get the question right, you want to activate two relays when siren output on DSC panel activates?
If so, just connect the 12V relay to the bell terminals on DSC panel. Place diode (such as 1N4148, 1N4007 or similar) parallel but in backwards to the relay.
